timeplot/__init__.py

```diff
@@ -0,0 +1,20 @@
+""" Plot event timelines for data exploration. Pass a pandas.DataFrame and a
```

Comment:
I see you have this documentation I was looking for here.  I think this would be more helpful by the other arguments' docstring entries. I'm guessing there is some utility to having the docstring right in the thing you import 'from timeplot import plot_events_timeline' but if that's true wouldn't the other parts of the docstring make sense here too?

You could also just merge the function from the other file in here and have one nice docstring. I'm not sure if you plan on adding a bunch of other stuff to this repo, but just having it all in timeplot/__init__.py (or even timeplot.py on the root level) wouldn't be the worst idea for this small of a package